lib/RoleIssuerResolver.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  DomainReader,
+  IIssuerDefinition,
+} from '@energyweb/credential-governance';
+import { IssuerResolver } from '@energyweb/vc-verification';
+import { providers, utils } from 'ethers';
+import { CacheServerClient } from './cacheServerClient';
+import { EthersProviderIssuerResolver } from '@energyweb/vc-verification';
+
+/**
+ * Resolves issuers definition by either by reading smart contract state via an Ethers provider or from ssi-hub (cache server)
```

Comment:
IMO it would be clearer this way
```suggestion
 * Resolves issuer definition from ssi-hub, if it is specified. Otherwise read issuer from blockchain
```